pom.xml

```diff
@@ -68,7 +69,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>docker-java-api</artifactId>
-            <version>3.0.14</version>
+            <version>3.1.5-rc11.59b91388afe6</version>
```

Comment:
We shouldn't be depending on a rc build.
We should only depend on a "proper" release.

Reply:
I know. I'm waiting for [this other PR](https://github.com/jenkinsci/docker-java-api-plugin/pull/5) to be merged. But it seems that maintainers of docker-java-api-plugin are all retired...

Reply:
FYI Nicolas De loof backed away from all-things-Jenkins some time ago.  We'd worked together on this plugin (after he'd added me as a co-maintainer) but it was a bit of a surprise for me to find I was "suddenly" the sole maintainer rather than co-maintainer of _this_ plugin.

Looking at the commit history for the `docker-java-api-plugin`, it looks like `olivergondza` has commit access; maybe he can help.  If not, there's a Jenkins process you can follow to get control of an abandoned plugin; you can become its maintainer and push changes through yourself.

(FYI this "maintainers are all retired" scenario is not unusual; of all the bits of Jenkins I've worked on, I've only avoided being given commit-access twice - "no good deed goes unpunished" :wink:)

Reply:
According to [this comment](https://github.com/jenkinsci/docker-java-api-plugin/pull/4#issuecomment-589619984), `olivergondza` is not a maintainer of docker-java-api-plugin.

I don't want to become a maintainer, I just want to contribute this feature.

It is scary that maintainers are giving up on Jenkins. Is Jenkins dead??

Reply:
Jenkins isn't dead, no; Jenkins is an open-source project and its life is independent of any single individual ... but not everyone who becomes involved in its development enough to become a contributor _stays_ involved forever, e.g. people change jobs.
Progress is mostly driven by folks who have a need to make changes and are prepared to put a bit of work in to make that happen; for some components, there's sometimes _years_ that go by without anyone needing to change anything and, during that time, whoever maintained it last may disappear off the radar.
That seems to be the case with the docker-java-api-plugin - it's been stable for so long it didn't need an active maintainer until you stirred things up 😛 

e.g. the vSphere plugin was started by one person, continued by another and, when I wanted to add to it, neither of those two people wanted any involvement with it anymore and basically told me "it's yours now!", which is how I became the maintainer of the vSphere plugin :wink:
When I wanted to add to the docker plugin, Nickolas was actively developing it and was happy to have a second contributor ... but then he left, leaving only me.
When I wanted to contribute to the OpenStack plugin, I was lucky - Oliver was happy to stay sole maintainer of that one.

FYI this is why the Jenkins project _overall_ has a "I want to contribute to this plugin but nobody's actively maintaining it anymore" process whereby an individual can get control of an abandoned plugin - it allows any given component to outlive the person who started it _without_ that person actively handing over controls to a successor.

As for becoming a maintainer, well, you don't have to do it forever (as Nicolas has demonstrated) - you take it over, get up to speed on it, drive this change to completion and then, once we're all happy that it's working and no further changes are needed, you can leave it without a maintainer - it's not exactly a high-traffic plugin.